### PR TITLE
update build plugins

### PR DESCRIPTION
**Pull request by pjfanning**

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

**Review by nvollmar — APPROVED** [review]

File: .github/workflows/integration-tests-kube-api.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
         uses: coursier/cache-action@v6
 
       - name: Setup Minikube
-        uses: manusa/actions-setup-minikube@v2.7.2
+        uses: manusa/actions-setup-minikube@v2
```
Comment (laglangyue) [inline .github/workflows/integration-tests-kube-api.yml:42]: maybe 
[v2.10.0](https://github.com/manusa/actions-setup-minikube/releases/tag/v2.10.0)

Comment (pjfanning) [reply .github/workflows/integration-tests-kube-api.yml:42]: this just avoids us having to update the build script every time there is a manusa/actions-setup-minikube release - we have this for most actions now

Comment (pjfanning) [reply .github/workflows/integration-tests-kube-api.yml]: changing due to the build issue

**Pull request merged**